ironfish-cli/src/utils/fees.ts

```diff
@@ -17,6 +17,34 @@ import {
 import inquirer from 'inquirer'
 import { promptCurrency } from './currency'
 
+export async function validateBalance(options: {
+  client: RpcClient
+  transaction: CreateTransactionRequest
+  account?: string
+  confirmations?: number
+  logger: Logger
+}): Promise<boolean> {
+  // check balance is sufficient for the minimal fee
+  const balanceRequest = await options.client.getAccountBalance({
+    confirmations: options.confirmations,
+  })
+
+  const balanceInOre = CurrencyUtils.decode(balanceRequest.content.available)
+  const currentAmountInOre = options.transaction.outputs.reduce(
+    (a, b) => a + CurrencyUtils.decode(b.amount),
+    0n,
+  )
+
+  if (balanceInOre <= 0 || balanceInOre <= currentAmountInOre) {
```

Comment:
This will solve the problem if the 'slow' fee rate is 1 ORE, but what if it's higher?

If the fee rate is 2 ORE and a user tries to send a transaction for their full balance minus 1 ORE will they still run into the same problem?

Reply:
updated and update the testing in description.